frontend/apps/reader/modules/readerhyphenation.lua

```diff
@@ -39,13 +39,20 @@ function ReaderHyphenation:init()
             local alg_left_hyphen_min = hyph_settings.left_hyphen_min
             local alg_right_hyphen_min = hyph_settings.right_hyphen_min
             local hyph_limits_widget = DoubleSpinWidget:new{
+                -- Min (2) and max (10) values are enforced by crengine
```

Comment:
OK, I must have written that false comment before some change I made to crengine :)
```c
// min value supported by algorithms is 1 (max is arbitrary 10)
// value enforced by algorithm previously was 2, so it's the default
#define HYPH_DEFAULT_HYPHEN_MIN 2
#define HYPH_MIN_HYPHEN_MIN 1
#define HYPH_MAX_HYPHEN_MIN 10
```
Can you please correct it, and just change `Min (2)` to `Min (1)`, so it's coherent with the `left_min = 1, right_min = 1,`?